pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -156,7 +156,15 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig) (*NodeConfig, error
 
 	// provide any config overrides
 	cfg.NodeName = options.NodeName
-	cfg.StreamingConnectionIdleTimeout = 5 * time.Minute // TODO: should be set
+	streamingConnectionIdletimeout, ok := options.KubeletArguments["streaming-connection-idle-timeout"]
+	if ok && len(options.KubeletArguments["streaming-connection-idle-timeout"]) > 1 {
```

Comment:
don't check flags directly... `cmdflags.Resolve(options.KubeletArguments, server.AddFlags)` above handles pushing flag values into the server's struct